src/Effects.js

```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.music_wave = {
+      wave: [],
+      init: function () {
+        let wave = {
+          inc: 360/15,
+          heightFactor: 1,
+          yHalf: 300,
+          colorUnit: 400,
+          colorHalf: 200,
+        };
+        this.wave.push(wave);
+      },
+
+      draw: function (context) {
+        const centroid = context.centroid;
+        let scale = p5.map(centroid, 5000, 8000, 0, 1);
+        let a = 0;
+        p5.background('black');
+        const wave = this.wave[0];
+        for (let i = 1; i <= wave.colorUnit; i+=5) {
```

Comment:
Is there a reason for starting this for-loop at `i=1` instead of `i=0`? In general, I think it's preferable to start at index 0.

Reply:
I'm not entirely sure what the problem is when I make `i=0`. When using the p5 editor it was fine, but when testing it locally `i=0` caused the wave to look drastically different and get cut off. That's why I left it as `i=1`.